src/blog/components/legal/LegalPreview.tsx

```diff
@@ -2,7 +2,7 @@
 import { QueryResponseInitial, useQuery } from "@sanity/react-loader";
```

Comment:
Remind me why we added `app/blog/components` ?

Reply:
I don't see that folder, are you talking about `src/blog/components`? If not, it might be a folder on your local machine that is not tracked by git (since git only tracks files).

Reply:
Sorry my bad, it looked like it was under app.

Still wondering why we've got `blog`?

Reply:
The folder has been there since the beginning, so you tell me 🙃

Reply:
Ahh right. So then this is placed in a wrong folder. 

blog was meant for UI specific blog and blogpost content.

Reply:
We'll fix this later.